### PR TITLE
Design and performance improvements

### PR DESCRIPTION
**Pull request by buzz-js**
## The ssr Property 

`mode` property change to `ssr` because mode option is deprecated.

ssr: false // for SPA's

## Buttons overlapping 

image

### Before
image

### After
image

### CODE REVIEW

**Comment by remidej** [comment]
Thanks for the contribution! I'll test your changes early next week

File: frontend/nuxt.config.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ export default {
    ** Nuxt rendering mode
    ** See https://nuxtjs.org/api/configuration-mode
    */
-  mode: 'universal',
+  ssr: true,
```
Comment (remidej) [inline frontend/nuxt.config.js:6]: I think we should make the site fully static, for better performance and easier deployments. Can you put `target: 'static'` here instead?

**Review by remidej — CHANGES_REQUESTED** [review]

File: frontend/nuxt.config.js (hunk 1 above)
Comment (buzz-js) [reply frontend/nuxt.config.js:6]: Right, I agree

Comment (remidej) [reply frontend/nuxt.config.js:6]: My bad, it's actually the way the data is fetched that breaks SSG. I'll fix this myself.

But I still think the `ssr: true` might conflict with SSG. Can you remove this sections altogether (delete lines 2 to 6) ?

Comment (buzz-js) [reply frontend/nuxt.config.js]: Done

**Pull request merged**